agent_issuance/src/credential/aggregate.rs

```diff
@@ -228,32 +261,47 @@ impl Aggregate for Credential {
 
                     info!("Credential: {:?}", credential);
 
-                    #[cfg(feature = "test_utils")]
-                    let iat = 0;
-                    #[cfg(not(feature = "test_utils"))]
-                    let iat = credential.raw["issuanceDate"]
+                    let issuance_date = credential.raw["issuanceDate"]
                         .as_str()
                         .unwrap()
                         .parse::<chrono::DateTime<chrono::Utc>>()
-                        .unwrap()
-                        .timestamp();
+                        .unwrap();
+
+                    #[cfg(feature = "test_utils")]
+                    let iat = 0;
+                    #[cfg(not(feature = "test_utils"))]
+                    let iat = issuance_date.timestamp();
                     // let iat = std::time::SystemTime::now()
                     //     .duration_since(std::time::UNIX_EPOCH)
                     //     .unwrap()
                     //     .as_secs() as i64;
 
+                    #[cfg(feature = "test_utils")]
+                    let exp = 0;
+                    #[cfg(not(feature = "test_utils"))]
+                    let exp = iat + 60 * 60 * 24 * 365; // TODO: currently hard-coded to one year from issuance date
```

Comment:
@nanderstabel Should be make this configurable? If yes, do you think it should be a config parameter or should it be supplied via the HTTP request?

Reply:
Probably both.

1. It think we should come up with our own default values for these type of variables, which can be overwritten by the user by:
2. Setting the default values for these variables in the `config.yaml`, which in turn can also be overwritten per Credential by:
3. Adding the values to the HTTP request.